polygeist/tools/cgeist/Test/Verification/raiseToAffineUnsignedCmp.c

```diff
@@ -21,4 +21,4 @@ void matmul(float A[100][200], float B[200][300], float C[100][300]) {
       }
     }
   }
-
+}
```

Comment:
Wow, how did this ever run ?

Reply:
Well, I guess the parser is smart and we were not failing on errors...